### PR TITLE
Monokle v1.3.0 feature update doc

### PR DESCRIPTION
**Pull request by iamrajeev**
Monokle v1.3.0 feature update doc

## Changes

-

## Fixes

-

## How to test it

-

## Screenshots

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: docs/features.md [hunk 1]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
 the cluster into Monokle - [read more](cluster-integration.md). If you need to make any immediate hot-fixes simply modify
 the resource in the source editor and click the "Apply" button.
 
+## Cluster Compare
 
+Cluster Compare functionality in Monokle 1.3.0 allows you to compare a set of local resources (plain or generated with Kustomize/helm) against a cluster. 
 
+![Cluster Compare](img/clustercompare-1.png)
 
+## The Improved Validation Logic and Error Display
```
Comment (olensmar) [inline docs/features.md:50]: remove "The"

File: docs/features.md [hunk 2]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
 
+In the latest Monokle 1.3.0, all the errors can be found at the top of the resource navigator to view the invalid resources.
```
Comment (olensmar) [inline docs/features.md:52]: "Since Monokle 1.3.0, all errors..."

File: docs/features.md [hunk 3]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
 
+Cluster Compare functionality in Monokle 1.3.0 allows you to compare a set of local resources (plain or generated with Kustomize/helm) against a cluster. 
 
+![Cluster Compare](img/clustercompare-1.png)
 
+## The Improved Validation Logic and Error Display
 
+In the latest Monokle 1.3.0, all the errors can be found at the top of the resource navigator to view the invalid resources.  
 
+![The Improved Validation Logic and Error Display](img/resource-navigator.png)
```
Comment (olensmar) [inline docs/features.md:54]: Remove "The"

File: docs/features.md [hunk 4]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
+
+Selecting a resource from the list will automatically open it for editing in the editor.  
+[The Improved Validation Logic and Error Display](img/resourcenavigator.gif)
+
+## Handling Multiple KubeConfig Contexts
+
+Supposedly, the chosen Kubernetes file contains numerous contexts. Monokle 1.3.0 will find those and allow you to select the right one for your cluster operations.
```
Comment (olensmar) [inline docs/features.md:61]: "If the selected kubeconfig file contains numerous contexts, Monokle will find those..."

File: docs/features.md [hunk 5]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
+
+## Handling Multiple KubeConfig Contexts
+
+Supposedly, the chosen Kubernetes file contains numerous contexts. Monokle 1.3.0 will find those and allow you to select the right one for your cluster operations. 
+
+![Handling Multiple KubeConfig Contexts](img/selectcontext.png)
+
+## Linux installer/Packages 
+
+The Monokle 1.3.0 includes several packages for different Linux distros.
```
Comment (olensmar) [inline docs/features.md:67]: Remove "The"

File: docs/features.md [hunk 6]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
+
+## Linux installer/Packages 
+
+The Monokle 1.3.0 includes several packages for different Linux distros. 
+
+![Linux installers/Packages](img/linuxpackages.png)
+
+## Improved Kustomize Support
+
+The Monokle 1.3.0 supports all types of kustomization files with or without the kind/api version. Moreover, it also offers an option to choose the command to kustomize the resources.
```
Comment (olensmar) [inline docs/features.md:73]: "..command to run Kustomize"

File: docs/features.md [hunk 7]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
+
+Monokle 1.3.0 does not show incorrect optional references as errors in resources. 
+
+![Managing Optional References](img/optionalreferences.png)
+
+As can be observed from the above snippet, configMapKeyRef has a value set to true - showing no error for the invalid for the name reference, in comparison to the one below which is set to false.  
+
+![Managing Optional References](img/optionalreferences2.png)
+
+You can always disable it in the Settings pane by selecting “Ignore Optional Unsatisfied Links.”
```
Comment (olensmar) [inline docs/features.md:87]: "You can disable this behavior in..."

File: docs/features.md [hunk 8]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
+
+![Managing Optional References](img/optionalreferences2.png)
+
+You can always disable it in the Settings pane by selecting “Ignore Optional Unsatisfied Links.” 
+
+![Managing Optional references](img/optionalreferences3.png)
+
+## “Unknown Resource Kind” Shown In Navigator 
+
+In Monokle 1.3.0, the unrecognized resources shown under a dedicated “Unknown” section are placed at the bottom of Navigator.
```
Comment (olensmar) [inline docs/features.md:93]: "In Monokle 1.3.0 unrecognized..."

File: docs/features.md [hunk 9]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
+
+## “Unknown Resource Kind” Shown In Navigator 
+
+In Monokle 1.3.0, the unrecognized resources shown under a dedicated “Unknown” section are placed at the bottom of Navigator. 
+
+![Unknown Resource Kind](img/unknown.png)
+
+## Notification Centre 
+
+In Monokle 1.3.0, a Notification center is added that allows you to have a quick view of all the past notifications.
```
Comment (olensmar) [inline docs/features.md:99]: ".. that gives you a quick view..."

File: docs/features.md [hunk 10]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
+Since Monokle 1.3.0, all the errors can be found at the top of the resource navigator to view the invalid resources.  
 
+![ Improved Validation Logic and Error Display](img/resource-navigator.png)
+
+Selecting a resource from the list will automatically open it for editing in the editor.  
+[The Improved Validation Logic and Error Display](img/resourcenavigator.gif)
+
+## Handling Multiple KubeConfig Contexts
+
+If the selected Kubernetes file contains numerous contexts, Monokle will find those and allow you to select the right one for your cluster operations.
```
Comment (olensmar) [inline docs/features.md:61]: "Kubeconfig file"

File: docs/features.md [hunk 11]
```diff
@@ -41,8 +41,61 @@ Want to browse your cluster instead of a repo? Simply smash the "Show Cluster Ob
...
+
+## Linux installer/Packages 
+
+Monokle 1.3.0 includes several packages for different Linux distros. 
+
+![Linux installers/Packages](img/linuxpackages.png)
+
+## Improved Kustomize Support
+
+The Monokle 1.3.0 supports all types of kustomization files with or without the kind/api version. Moreover, it also offers an option to choose the command to run kustomize.
```
Comment (olensmar) [inline docs/features.md:73]: Remove "The.."

File: docs/features.md (hunk 10 above)
Comment (iamrajeev) [reply docs/features.md]: Done

File: docs/features.md (hunk 11 above)
Comment (iamrajeev) [reply docs/features.md]: Done

**Review by olensmar — APPROVED** [review]

**Pull request closed without merging**